tools/find_dali.cmake

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+####################################################################
+# Utility function, useful when building custom C++ library where
+# DALI is used as a shared library. Assumes that DALI wheel is
```

Comment:
If it assumes that DALI is installed it should be added to the wheel.

Reply:
Actually, I'm not convinced, that is should. This is more of an example, we don't use it anywhere in our code (except for test) and I think we don't include examples in the whl. Also, it doesn't make much sense, without `CMakeLists.txt` file, which I guess we don't have in the whl. But if you feel it should be there, I'll add it, no problem

Reply:
Then the whole logic doesn't make sense. How it suppose to be a CMake only utility as you expect to have wheel installed.
If it is a part of the source then it should not expect wheel in the system.

Reply:
The purpose of this function is that user copy-pastes it into his CMake routines and uses it to generate DALI_LIB_DIR, DALI_INCLUDE_DIR and DALI_LIBRARIES. This is kind of poor-man, python-based `find_package(DALI)`. We would like to have it in the CI, to be sure that this method still works (and if it doesn't, we will know it). That's how DALI's used in the C++ app right now